### DISCUSSION TITLE
Can I disable sddm and just use a terminal for install?

### DISCUSSION THREAD

**Discussion by Kazpa** [prasanthrangan/hyprdots#906] [Q&A]
I used to use etc/issue as a customized login screen a while back for my Hyprland set up but with this current setup, we all seem to use sddm. I don't like any of the sddm themes personally and just want a bare bones login that displays acii art I created. Can I disable sddm without worrying about crashing and go back to using etc/issue or an altnerative? Thanks and wicked awesome theme you have here!

**Comment by kRHYME7** [comment]
Try to disable it I think It's fine. I'm using getty to login.

systemctl disable sddm.service
